src/ceph-daemon/ceph-daemon

```diff
@@ -1622,6 +1625,12 @@ def _get_parser():
         '--verbose', '-v',
         action='store_true',
         help='Show debug-level log messages')
+
+    parser.add_argument(
+        '--devel', '-d',
```

Comment:
:-1: here. Especially `ceph-daemon -d` looks like it would aid debugging for end users. If we really need a dedicated flag, I would be more explicit: `cpeh-daemon --keep-tmp-files`.

Not sure if it's worth maintaining this log term.

Reply:
yep, agree ...